mw-webapp/src/component/progressBar/ProgressBar.module.scss

```diff
@@ -0,0 +1,22 @@
+@import "src/styles/variables";
+
+$progressBarContainerHeigh: 20px;
+
+.progressBarContainer {
+  height: $progressBarContainerHeigh;
```

Comment:
missing 't' letter in the  word Height (progressBarContainerHeigh**t**)

Reply:
Done